CONTRIBUTING.md

```diff
@@ -446,17 +462,13 @@ This project is an open source project, please act responsibly, be nice, polite
 
 * The Maven build fails with `OutOfMemoryException`
 
-  Set Maven options to use 1.5GB of heap: `export MAVEN_OPTS="-Xmx1563m"`.
+  Set Maven options to use more memory: `export MAVEN_OPTS="-Xmx4g"`.
```

Comment:
With 4g now set in `.mvn/jvm.config`, the example here doesn't really change anything, does it?

Reply:
True - and technically I think we could remove the section.  But I did wonder if someone has a `MAVEN_OPTS` set, it might override the project settings and get you in trouble - which would be fixed by gibing it "more memory" and setting this explicitly.

Reply:
Yes, `MAVEN_OPTS` should override this but my main point is/was that 4 is less than 5. 😉